builder-support/debian/authoritative/debian-buster/config/ldap.conf

```diff
@@ -1,7 +1,77 @@
-# LDAP Configuration
+# See https://doc.powerdns.com/authoritative/backends/ldap.html
+launch+=ldap
+
+#################################
+# ldap-basedn	Search root in ldap tree (must be set)
 #
-# ldap-host=
-# ldap-method=
 # ldap-basedn=
+
+#################################
+# ldap-basedn-axfr-override	Override base dn for AXFR subtree search
+#
+# ldap-basedn-axfr-override=no
+
+#################################
+# ldap-binddn	User dn for non anonymous binds
+#
 # ldap-binddn=
+
+#################################
+# ldap-bindmethod	Bind method to use (simple or gssapi)
+#
+# ldap-bindmethod=simple
+
+#################################
+# ldap-disable-ptrrecord	Deprecated, use ldap-method=strict instead
+#
+# ldap-disable-ptrrecord=no
+
+#################################
+# ldap-filter-axfr	LDAP filter for limiting AXFR results
+#
+# ldap-filter-axfr=(:target:)
+
+#################################
+# ldap-filter-lookup	LDAP filter for limiting IP or name lookups
+#
+# ldap-filter-lookup=(:target:)
+
+#################################
+# ldap-host	One or more LDAP server with ports or LDAP URIs (separated by spaces)
```

Comment:
```suggestion
# ldap-host	One or more LDAP servers with ports or LDAP URIs (separated by spaces)
```

Reply:
this is auto-generated from pdns code, please fix it there first, or it will be reintroduced.